autogen/main/variables.tf.tmpl

```diff
@@ -833,3 +833,11 @@ variable "enable_gcfs" {
 }
   {% endif %}
 {% endif %}
+
+{% if autopilot_cluster %}
+variable "allow_net_admin" {
+  description = "(Optional) Enable NET_ADMIN for the cluster."
+  type = bool
+  default = false
```

Comment:
Would it be better to default to `null` to limit impact to existing clusters?

Reply:
Good catch - thanks! I'll make that change now